package-lock.json

```diff
@@ -13,7 +13,7 @@
     "@babel/core": {
       "version": "7.5.5",
       "resolved": "https://registry.npmjs.org/@babel/core/-/core-7.5.5.tgz",
-      "integrity": "sha512-i4qoSr2KTtce0DmkuuQBV4AuQgGPUcPXMr9L5MyYAtk06z068lQ10a4O009fe5OB/DfNV+h+qqT7ddNV8UnRjg==",
+      "integrity": "sha1-F7JobvDWvFj5Y93daKtml1VYLDA=",
```

Comment:
@alanmoo do you know why there are changes in `integrity`?

Reply:
Good catch. It seems like sha1 is an older hashing method, I'd rather not downgrade the validation of our packages. Looks like https://www.dnasir.com/2018/02/12/solving-the-package-lock-json-integrity-discrepancy/ has some tips on addressing it.

Reply:
@alanmoo Thanks! 

> Step 1. Ensure all team members use the same Node and NPM versions.

This is gonna be hard 😂

Reply:
@Sharma-palak since your PR didn't introduce any npm module changes, can you uncommit `package-lock.json` from this PR so we can just stick with the current `package-lock.json` we have on `master`?

Reply:
ok @mmmavis  @alanmoo

Reply:
@mmmavis install `nvm` globally so you can switch node/npm versions as needed with a simple `nvm use v12.6.0` or similar command.